eco-heron-examples/src/java/org/apache/heron/examples/eco/EvenAndOddBolt.java

```diff
@@ -0,0 +1,76 @@
+//  Copyright 2018 Twitter. All rights reserved.
```

Comment:
Quick questions:

"Copyright 2018 Twitter. All rights reserved."

1. Do we need to change this to Apache as well?
2. How about the year number?

Reply:
i want to keep the any other changes (including copyright that you suggested) separate from package renaming.
also, i am waiting to hear back from Mentors, if a long single would be convenient or the current way.